frontend/src/router/index.ts

```diff
@@ -51,7 +51,7 @@ const routes = [
         component: LandingView,
     },
     {
-        path: routeItems.dashboard.path,
+        path: `${routeItems.dashboard.path}/:newUserInTable?`,
```

Comment:
The 'newUserInTable' name does not make sense.
However, this is very interesting, what do you mean by `/:newUserInTable?`? What do you expect it to happen? Is `:newUserInTable` a placeholder? Then what about the "?" mark at the end?

Does it show up on browser address bar (like /123?) ? I can see it didn't and that puzzles me, is it some feature from vue-router I didn't notice? But if you look closely the browser behavior, it actually did it twice for routing. Maybe I misunderstand something in vue-router...

Reply:
The ` /:newUserInTable?` is a placeholder and the "?" means its optional. About it doing twice I'm not sure, I'll see what I can find.

Reply:
Always add **clear comment** for this kind of odd looking string (if it was provided by vue-router doc, better give a reference to it), so other people would not be puzzled for that.

Please look into why it routed twice, if you see it triggers fetching twice, I would not feel good for it (I would not be surprised it is a bug from vue router if it was provided such kind of syntax but fetching twice).